includes/payment-methods/class-wc-stripe-express-checkout-ajax-handler.php

```diff
@@ -140,6 +140,21 @@ public function ajax_clear_cart() {
 		exit;
 	}
 
+	/**
+	 * Normalizes address fields in WooCommerce supported format.
+	 */
+	public function ajax_normalized_address() {
```

Comment:
nit: rename function to match current convention
```suggestion
	public function ajax_normalize_address() {
```